HunterPie.Core/Core/Local/Player.cs

```diff
@@ -1262,6 +1269,22 @@ private void UpdateHarvestBoxCounter(long LastFertAddress)
             Harvest.Counter = counter;
         }
 
+        private int GetHarvestBoxSize()
+        {
+            var noob = Kernel.Read<long>(Address.GetAddress("BASE") + Address.GetAddress("NOOB_OFFSET"));
+            if (Kernel.Read<char>(noob + 0x4d3) == 0x1)
+            {
+                return 10;
+            }
```

Comment:
@Haato3o What if we don't need this check? Like what if it's just a way to short circuit the bit counting if it's too early for harvest box upgrades? Would you be comfortable if I removed this? It's probably not needed. I imagine the bits we're checking will be 0 if it's too early for the harvest box, so `(0 + 1) * 10` would yield the same result.

This way I don't need to rename NOOB_OFFSET because we don't need it.